packages/manager/modules/iam/src/policies/myPolicies/myPolicies.routing.js

```diff
@@ -13,11 +13,29 @@ export default /* @ngInject */ ($stateProvider) => {
         type: 'cursors',
         value: null,
       },
+      identitiesFilter: {
+        array: true,
+        value: [],
+      },
+      resourcesFilter: {
+        array: true,
+        value: [],
+      },
+      actionsFilter: {
+        array: true,
+        value: [],
+      },
```

Comment:
You can keep the same names than the ones used by the API
```suggestion
      identities: {
        array: true,
        value: [],
      },
      resources: {
        array: true,
        value: [],
      },
      actions: {
        array: true,
        value: [],
      },
```